lib/ResolverFactory.js

```diff
@@ -280,7 +280,7 @@ exports.createResolver = function (options) {
 
 	resolver.ensureHook("resolve");
 	resolver.ensureHook("internalResolve");
-	resolver.ensureHook("newInteralResolve");
+	resolver.ensureHook("newInternalResolve");
```

Comment:
Potential breaking change, better to add deprecation

Reply:
How would a depreciation work for this one?
PS: did a quick search and it doesn't look like it's referenced anywhere else https://github.com/search?q=org%3Awebpack+Interal&type=code

Reply:
This package used not only in webpack, many tools use it for resolving